capellambse/model/__init__.py

```diff
@@ -324,3 +328,171 @@ def search(
+
+    # pylint: disable=line-too-long
+    def update_diagram_cache(
+        self,
+        capella_cli: str,
+        image_format: t.Literal["bmp", "gif", "jpg", "png", "svg"] = "svg",
+        create_index: bool = False,
+        force: t.Literal["exe", "img"] | None = None,
+    ) -> None:
+        """Update the diagram cache if one has been specified.
+
+        If a diagram cache has been specified during the load of a
+        Capella model it will be updated when this function is called.
+        The diagram cache will be populated by executing the Capella
+        function "Export representations as images" which is
+        normally accessible via the context menu of an ``.aird`` node in
+        Capella's project explorer. The export of diagrams happens with
+        the help of Capella's command line interface (CLI).
+        The CLI of Capella must be specified by the caller. It is
+        possible to work with a local installation or a Docker image of
+        an individual Capella bundle.
+        At the moment it is supported to run a Docker image using the
+        container system Docker and the ``docker`` executable must be in
+        the ``PATH`` environment variable.
+
+        Parameters
+        ----------
+        capella_cli
+            The Capella CLI to use when exporting diagrams from the
+            given Capella model. The provided string can come with a
+            ``"{VERSION}"`` placeholder. If specified, this placeholder
+            will be replaced by the x.y.z formatted version of Capella
+            that has been used when the given Capella model was last
+            saved. After consideration of the optional placeholder this
+            function will first check if the value of *capella_cli*
+            points to a local Capella executable (that can be an
+            absolute path or an executable/ symbolic link that has been
+            made available via the environment variable ``PATH``). If no
+            executable can be found it is expected that *capella_cli*
+            represents a Docker image name for an image that behaves
+            like the Capella CLI. For the case of passing a Docker image
+            name through *capella_cli* this means it is assumed that
+            something like
+
+            .. code-block:: bash
+
+                docker run --rm -it <capella_cli> -nosplash -consolelog -app APP -appid APPID
+
+            will work.
+            The parameter *force* can be set to change the described
+            behaviour and force the function to treat the *capella_cli*
+            as a local executable or a Docker image only.
+        image_format
+            Format of the image file(s) for the exported diagram(s).
+            This can be set to any value out of ``"bmp"``, ``"gif"``,
+            ``"jpg"``, ``"png"``, or ``"svg"``.
+        create_index
+            If true two index files ``index.json`` and ``index.html``
```

Comment:
"True" in this case is a Python keyword, not an English word. Never alter keywords or names in documentation.

Reply:
Well, I can live with that. At the same time I'd say that keywords must be formatted so that we'll see:

```text
If ``True``, two index files (...)
```